tests/openshift/pipelines/E2E-Test-JenkinsFile.groovy

```diff
@@ -2,7 +2,7 @@
 podTemplate(label: 'bddstack', name: 'bddstack', serviceAccount: 'jenkins', cloud: 'openshift', containers: [
   containerTemplate(
     name: 'jnlp',
-    image: 'docker-registry.default.svc:5000/zwmtib-tools/bddstack-node:2.0',
+    image: 'docker-registry.default.svc:5000/vmvfjv-tools/bddstack',
```

Comment:
We should consider having the namespace portion be parameterized if possible down the line.

Reply:
Agree - POC for now